opencsp/common/lib/csp/RayTrace.py

```diff
@@ -572,8 +792,33 @@ def plane_intersect_OLD(
     ray_trace: RayTrace, v_plane_center: Vxyz, u_plane_norm: Uxyz, epsilon: float = 1e-6, verbose=False
 ) -> Vxy:
     """
-    TODO This was the old way of doing plane intersections. Can probably be removed safely.
+    Calculates the intersection points of light paths with a specified plane.
+
+    This method is deprecated and may be removed in future versions.
+
+    Parameters
+    ----------
+    ray_trace : RayTrace
+        The RayTrace object containing the light paths.
+    v_plane_center : Vxyz
+        The center point of the plane.
+    u_plane_norm : Uxyz
+        The normal vector of the plane.
+    epsilon : float, optional
+        The threshold for determining if a ray is parallel to the plane (default is 1e-6).
+    verbose : bool, optional
+        If True, prints execution status (default is False).
+
+    Returns
+    -------
+    Vxy
+        The intersection points in the local plane XY reference frame.
+
+    Notes
+    -----
+    This method is deprecated. Use the `plane_intersect` function instead.
     """

```

Comment:
IDK, possibly same thing here (I'll leave it up to you). Since this has "OLD" in method name and this isn't used anywhere else in the repo, maybe it's not worth documenting and we should just delete it. But maybe that's for another PR.